meta-facebook/yv3-dl/src/platform/plat_sensor_table.c

```diff
@@ -366,3 +366,8 @@ void pal_extend_sensor_config()
 
 	return;
 }
+
+uint8_t get_hsc_pwr_reading(int *reading)
+{
+	return get_sensor_reading(SENSOR_NUM_HSC_PIN, reading, GET_FROM_CACHE);
+}
```

Comment:
Check to make sure reading isn't null (There is a macro for this I added)

Reply:
Modify as recommendation.